package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@netzstrategen/gulp-task-collection",
-  "version": "4.0.0",
+  "version": "5.0.0",
```

Comment:
Is this a breaking change to previous versions?  If not, this should be 4.1.0.

Reply:
Shouldn't be. I will change a couple of things to make it a minor.